pkgs/top-level/all-packages.nix

```
@@ -25838,7 +25838,8 @@ in
 
   gap-full = lowPrio (gap.override { packageSet = "full"; });
 
-  geogebra = callPackage ../applications/science/math/geogebra { };
+  geogebra = callPackage ../applications/science/math/geogebra { };	
```

Remove trailing space
```suggestion
  geogebra = callPackage ../applications/science/math/geogebra { };
```